src/Navbar.jsx

```diff
@@ -1,36 +1,44 @@
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
+import { IconButton, Button } from '@chakra-ui/react';
+import {ChevronLeftIcon} from '@chakra-ui/icons';
+import { useNavigate } from 'react-router-dom';
 
-function Navbar({ isAdmin }) {
+// Navbar is a container with a single button for Contact Us
+
+function Navbar({ showContactUs, title, showSettings, excelDownload, backButton}) { // what is title
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/latest/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'title' is defined but never used.